src/amo/components/Addon/index.js

```diff
@@ -292,14 +306,17 @@ export class AddonBase extends React.Component {
     const addonPreviews = addon ? addon.previews : [];
 
     let isCompatible = false;
+    let isFeatured = false;
     let compatibility;
     if (addon) {
       compatibility = getClientCompatibility({
         addon, clientApp, userAgentInfo,
       });
       isCompatible = compatibility.compatible;
+      isFeatured = addon.is_featured;
```

Comment:
I know this file isn't using Flow but could you add `is_featured` to the [Addon type](https://github.com/willdurand/addons-frontend/blob/cb3c68935ee7b2fe966660fdb7f55ef88d2912c1/src/core/types/addons.js#L39) while you're in here?